scripts/update-versions.sh

```diff
@@ -9,6 +9,9 @@ versioned_packages=(
   "plutus-tx-plugin"
 )
 
+additional_packages=( "$@" )
+versioned_packages+=( "${additional_packages[@]:1}" )
```

Comment:
I was actually thinking something more like:
```
version=$1
shift
default_packages=...
if [ "$#" == "0" ]; then
   packages=default_packages
else
   packages=( "$@" )
```
That way if I've run `update-versions` already and I just need to do some specific ones then I can just do exactly those. Consider:
```
update-versions 1.2
update-versions 1.0.1 prettyprinter-configurable
# oops, updated plutus-tx-plugin to 1.0.1 now :(
```

Reply:
Ok, I guess we can make the script work that way. But in terms of versioning, my preference is to use the same version for everything, although we can skip releasing a version if it hasn't changed. Versioning packages independently makes everything more complicated without much benefit.

Reply:
Oh yes, good point. I think that's fine then. I just think it's a nicer factoring to have effectively both `updateSpecificVersions` and `updateDefaultVersions`, but yeah, what's here is fine.